packages/orchestrator-mcp/src/specialization-system.ts

```
@@ -0,0 +1,1270 @@
+  }
+
+  /**
+   * Check certification requirements
+   */
+  private checkCertificationRequirements(profile: ToolProfile, program: CertificationProgram): boolean {
+    for (const req of program.requirements) {
+      switch (req.type) {
+        case 'tasks':
+          const taskCount = profile.performanceHistory.filter(r =>
+            r.domain === program.domain
+          ).length;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)